nexus/networking/src/firewall_rules.rs

```diff
@@ -345,6 +353,45 @@ pub async fn resolve_firewall_rules_for_sled_agent(
             priority: rule.priority.0 .0,
         });
     }
+
+    // Add rules that implement our IP allowlist for user-facing services.
+    //
+    // These rules are implicit, and not stored in the firewall rule table,
+    // since they're logically a different thing. We'll convert them into
+    // sled-agent firewall rules here and insert them.
+    //
+    // Note that this only applies to the internal services VPC, so we ignore
+    // these rules if we have no such interfaces for that.
+    if vpc.id() == *SERVICES_VPC_ID {
+        if let AllowedSourceIps::List(allowed_ips) =
```

Comment:
I'd write this as a `match` instead if `if let / else` so if we add new variants, we're forced to update this. (Otherwise new variants would fall into the "allow any" case, which seems bad!)

Reply:
Done in 381b1d2c0